src/flow2supera/reader.py

```diff
@@ -122,7 +122,7 @@ def GetNeutrinoIxn(self, ixn, ixn_idx):
         nu_result.id = int(ixn_idx)
         nu_result.interaction_id = int(ixn['vertex_id']) 
         nu_result.target = int(ixn['target'])
-        nu_result.vtx = supera.Vertex(ixn['vertex'][0], ixn['vertex'][1], ixn['vertex'][2], ixn['vertex'][3])
+        nu_result.vtx = supera.Vertex(ixn['x_vert'], ixn['y_vert'], ixn['z_vert'], ixn['t_vert'])
```

Comment:
Keep this local for now/or in a separate branch until we know for sure the variable name is going to change in the next production